metrics/error-report/Bugsnag.js

```diff
@@ -54,12 +64,19 @@ function initClient(sourceId) {
     appVersion: "process.package_version", // Set SDK version as the app version
     metadata: META_DATA,
     onError: (event) => {
-      const errorOrigin = event.errors[0].stacktrace[0].file;
-      if (typeof errorOrigin === "string") {
-        const index = errorOrigin.lastIndexOf("/");
-        if (!SDK_FILE_NAMES.includes(errorOrigin.substring(index + 1)))
+      const errorOrigin = event.errors[0]?.stacktrace[0];
```

Comment:
Optional chaining syntax is currently broken in v1.1 SDK. Let's use the `get` utility instead.

Reply:
Addressed.